Code/skyrim_ui/src/app/components/group/group.component.ts

```diff
@@ -125,24 +125,54 @@ export class GroupComponent implements OnInit, OnDestroy {
   }
 
   private onPartyAutoHideState() {
-    this.settings.partyAutoHideChange
+    this.settings.autoHideParty
       .pipe(takeUntil(this.destroy$))
       .subscribe((state: boolean) => {
         this.isAutoHide.next(state);
-        if (this.settings.isPartyShown()) {
+        if (this.settings.isPartyShown.value) {
           this.isShown.next(true);
         }
         this.flashGroup();
       });
   }
 
+  private onPositionUpdate() {
+    combineLatest([this.settings.partyAnchorOffsetX, this.settings.partyAnchorOffsetY, this.settings.partyAnchor])
+    .pipe(takeUntil(this.destroy$))
+    .subscribe( ([x, y, anchor]) => {
+      const newPosition: GroupPosition = {};
+
+      switch (anchor) {
+        case PartyAnchor.TOP_LEFT:
+          newPosition.top = `${ y }vh`;
+          newPosition.left = `${ x }vw`;
+          break;
+        case PartyAnchor.TOP_RIGHT:
+          newPosition.top = `${ y }vh`;
+          newPosition.right = `${ -100 + x }vw`;
+          break;
+        case PartyAnchor.BOTTOM_RIGHT:
+          newPosition.bottom = `${ -100 + y }vh`;
+          newPosition.right = `${ -100 + x }vw`;
+          break;
+        case PartyAnchor.BOTTOM_LEFT:
+          newPosition.bottom = `${ -100 + y }vh`;
+          newPosition.left = `${ x }vw`;
+          break;
+      };
+      this.positionStyle.next(newPosition);
+    })
+  }
+
   ngOnDestroy() {
     if (this.timerSubscription) {
       this.timerSubscription.unsubscribe();
     }
   }
 
   private flashGroup() {
+    const timerLength = parseInt(this.settings.autoHideTime.value) * 1000;
```

Comment:
As discussed, this variable would be better suited being closer to where its used.